modules/evm-bridge/src/lib.rs

```diff
@@ -212,10 +213,7 @@ impl<T: Config> Pallet<T> {
 		// the corresponding parameter or return value.
 		// - part 2: 32 byte, string length
 		// - part 3: string data
-		ensure!(
-			output.len() >= 64 && output.len() % 32 == 0,
-			Error::<T>::InvalidReturnValue
-		);
+		ensure!(output.len() >= 64, Error::<T>::InvalidReturnValue);
```

Comment:
The above comments say output is 32-byte aligned, not anymore?

Reply:
revert